src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
+
+    private:
```

Comment:
>It's not specifically addressed in the coding guidelines

It's not an exhaustive list you know... probably should add in it.

>but many other header files seem to have separate private: blocks for function and member variable declarations.

hmmm, I don't feel the same way.... anyway, at here, there is no point doing so.